modules/salt/pages/yomi.adoc

```diff
@@ -0,0 +1,176 @@
++
+----
+mkdir -p venv/etc/salt/pki/{master,minion}
+----
+.  Create two new virtual environments:
++
+----
+venv/etc/salt/autosign_grains
+venv/var/cache/salt/master/file_lists/roots
+----
+. Add the configuration information to the new location:
++
+----
+cat <<EOF > venv/etc/salt/master
+root_dir: $(pwd)/venv
+
+file_roots:
+  base:
+    - $(pwd)/srv/salt
+
+pillar_roots:
+  base:
+    - $(pwd)/srv/pillar
+EOF
+----
+
+
+
+== Install the Yomi Formula Package
+
+
+You can use the Yomi formula to install the states and modules you need.
+
+The formula installs the states in [path]``/usr/share/salt-formulas/states``, and configuration files in [path]``/usr/share/yomi``.
+It also stores some pillar examples in [path]``/usr/share/yomi/pillar``.
+
+.Procedure: Installing the Yomi Formula
+
+. On the {productname} Server, at the command prompt, as root, install the formula:
++
+----
+zypper in yomi-formula
+----
+. So that ``salt-master`` can find the pillar, change the
+``pillar_roots`` entry in the configuration file, or use the one provided by the package:
++
+----
+cp -a /usr/share/yomi/pillar.conf /etc/salt/master.d/
+----
+. Restart services:
++
+----
+systemctl restart salt-master.service
+----
+
+== Prepare the Salt Pillar
+
+The ``salt-api`` service is required so that Yomi can monitor the installation.
+
+.Procedure: Preparing the Salt Pillar
+
+. Enable this service:
++
+----
+cp /usr/share/yomi/salt-api.conf /etc/salt/master.d/
+systemctl restart salt-master.service
+----
+. OPTIONAL: Open the [path]``/etc/salt/master.d/salt-api.conf`` configuration file, and add the required certificates to enable SSL.
+. Open the pillar configuration file (location?) and add or edit these lines:
++
+----
+config:
+  events: yes
+----
+
+
+When the pillar is configured, you can launch the ``monitor`` tool from the command prompt.
+The `monitor` tool storez authentication tokens generated by Salt API in a local cache.
+This makes future connections to the service simpler, but can sometimes can cause authentication errors.
+If you start the ``monitor`` tool with the ``-r`` option, then the local cache is removed before the connection is made, which can avoid authentication problems.
+
+Launch the ``monitor`` tool:
++
+----
+export SALTAPI_URL=http://localhost:8000
+export SALTAPI_EAUTH=file
+export SALTAPI_USER=salt
+export SALTAPI_PASS=linux
+
+monitor -r -y
+----
+
+[[WARNING]]
+====
+In this example, the password is in plain text.
+Do not use this in a production environment.
+====
+
+
+
+== Boot Clients
+
+Yomi is a set of Salt states that are used to drive the installation of a new operating system.
+To take full control of the system where the installation will be done, you will need to boot from an external system that provides an already configured `salt-minion`, and a set of CLI tools required during the installation.
+
+You can install YOMI on clients using different mechanisms.
```

Comment:
```suggestion
You can install Yomi on clients using different mechanisms.
```
Or use YOMI everywhere (better not, more than 3 uppercase letters are not that user-friendly).